library/node

```diff
@@ -4,6 +4,36 @@ Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@n
 GitRepo: https://github.com/nodejs/docker-node.git
 GitFetch: refs/heads/main
 
+Tags: 18-alpine3.14, 18.0-alpine3.14, 18.0.0-alpine3.14, alpine3.14, current-alpine3.14
+Architectures: amd64, arm32v6, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 2b67149d782e4716b9c51f348b401e9d07940831
+Directory: 18/alpine3.14
+
+Tags: 18-alpine, 18-alpine3.15, 18.0-alpine, 18.0-alpine3.15, 18.0.0-alpine, 18.0.0-alpine3.15, alpine, alpine3.15, current-alpine, current-alpine3.15
+Architectures: amd64, arm32v6, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 2b67149d782e4716b9c51f348b401e9d07940831
+Directory: 18/alpine3.15
+
+Tags: 18, 18-bullseye, 18.0, 18.0-bullseye, 18.0.0, 18.0.0-bullseye, bullseye, current, current-bullseye, latest
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 2b67149d782e4716b9c51f348b401e9d07940831
+Directory: 18/bullseye
+
+Tags: 18-bullseye-slim, 18-slim, 18.0-bullseye-slim, 18.0-slim, 18.0.0-bullseye-slim, 18.0.0-slim, bullseye-slim, current-bullseye-slim, current-slim, slim
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 2b67149d782e4716b9c51f348b401e9d07940831
+Directory: 18/bullseye-slim
+
+Tags: 18-buster, 18.0-buster, 18.0.0-buster, buster, current-buster
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 2b67149d782e4716b9c51f348b401e9d07940831
+Directory: 18/buster
+
+Tags: 18-buster-slim, 18.0-buster-slim, 18.0.0-buster-slim, buster-slim, current-buster-slim
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 2b67149d782e4716b9c51f348b401e9d07940831
+Directory: 18/buster-slim
+
 Tags: 17-alpine3.14, 17.9-alpine3.14, 17.9.0-alpine3.14, alpine3.14, current-alpine3.14
```

Comment:
This one is no longer the leading "alpine3.14" and "current-alpine3.14", that's Node 18 now.

Reply:
yeah, https://github.com/nodejs/docker-node/pull/1697#issuecomment-1103483219